### DISCUSSION TITLE
Neural Compute Stick 2 (NCS2) - Any use feedback?

### DISCUSSION THREAD

**Discussion by FrigateEnthusiast** [blakeblackshear/frigate#8892] [General]
I have a Intel Compute Stick 2 (NCS2) sitting in a drawer collecting dust and was considering using it in a one-off Frigate install on maybe a PI. Does anyone have any real world feedback, such as inference speed, quirks, etc. ??

**Comment by FrigateEnthusiast** [comment]
Pi project would be for the wife's birdhouse. If "bird" is detected all is cool. If squirrel is detected, trigger quick 10 round burst from fully automatic bb gun... LOL

**Comment by NickM-27** [comment]
The docs show inference speeds and notes about the NCS2. It is worth noting however that it has been discontinued by Intel and their newer OpenVINO libraries do not support it so at some point frigate may also not support it.

**Reply by NickM-27** [reply]
https://docs.frigate.video/frigate/hardware#openvino
